### DISCUSSION TITLE
KusionStack + KCL 17th Weekly Meeting | 第 17 次周会 (2023.07.06)

### DISCUSSION THREAD

**Discussion by Peefy** [kcl-lang/community#2] [General]
KusionStack 社区成员介绍、开源之夏、GLCC 内容同步、近期开发日常工作同步

Peefy: KCL Logo 征集
Peefy: KRM KCL 介绍
腾讯会议: 2023/07/06 20:00-20:45 (GMT+08:00) 中国标准时间 - 北京

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/oeO4dCUJeZCk

#腾讯会议：767-333-824

复制该信息，打开手机腾讯会议即可参与

钉钉会议
842 251 09536

**Comment by DonaldCrowley52** [comment]
Deze synchronisatiesessie leek erg rijk en inhoudelijk. Er waren introducties van communityleden, discussies over Summer of Open Source en GLCC, waardoor het zowel voor nieuwe als bestaande leden toegankelijk was. Peefy's presentatie over de KCL-logo-ontwerpwedstrijd en de introductie van KRM KCL zullen zeker in de smaak vallen bij geïnteresseerden in KusionStack en het KCL-ecosysteem. De synchronisatie met recente ontwikkelingen en dagelijkse werkzaamheden [casino Nalu](https://nalucasino.nl/) gaf ook een duidelijker beeld van de huidige voortgang en richting van het project. De planning was bovendien gunstig, ideaal om na het werk deel te nemen. Al met al voelde het als een bijeenkomst gericht op uitwisseling en kennisdeling, en niet alleen op eenrichtingscommunicatie, en perfect voor wie zich verder in de community wil verdiepen.

**Comment by elenascljha** [comment]
Get ready for fast-paced fun and big wins at QuickWin Casino! We offer a thrilling selection of casino games, including the latest slots, popular table games, and a dynamic live casino experience with professional dealers. QuickWin is designed for speed and simplicity, with a user-friendly interface ensuring effortless navigation and an immersive gaming experience. Visit [https://quickwin-casino.net/](https://quickwin-casino.net/) to discover our exciting promotions, rewarding bonus programs, and extensive game library. QuickWin Casino is committed to providing a secure and entertaining atmosphere – the ideal choice for your next online gaming adventure. Join QuickWin today and experience the thrill of instant wins!
